tests/unit/unit_test_synapseutils_sync.py

```diff
@@ -121,3 +124,50 @@ def test_syncFromSynapse__non_file_Entity():
          patch.object(syn, "get", return_value = Schema(name="asssdfa", parent="whatever")):
         assert_raises(ValueError, synapseutils.syncFromSynapse, syn, table_schema)
 
+
+def test_write_manifest_data__unicode_characters_in_rows():
+    # SYNPY-693
+    if not pandas_available:
```

Comment:
This feature does not depend on pandas, the tests should not be depended on pandas. 
Our build system should make sure all tests passed. If pandas are not available, tests should fail.